bitnami/harbor/values.yaml

```
@@ -3701,6 +3716,251 @@ trivy:
+  ##
+  livenessProbe:
+    enabled: true
+    initialDelaySeconds: 20
+    periodSeconds: 10
+    timeoutSeconds: 5
+    failureThreshold: 6
+    successThreshold: 1
+  ## Harbor Exporter containers' readiness probe
+  ## ref: https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#container-probes
+  ## @param exporter.readinessProbe.enabled Enable readinessProbe
+  ## @param exporter.readinessProbe.initialDelaySeconds Initial delay seconds for readinessProbe
+  ## @param exporter.readinessProbe.periodSeconds Period seconds for readinessProbe
+  ## @param exporter.readinessProbe.timeoutSeconds Timeout seconds for readinessProbe
+  ## @param exporter.readinessProbe.failureThreshold Failure threshold for readinessProbe
+  ## @param exporter.readinessProbe.successThreshold Success threshold for readinessProbe
+  ##
+  readinessProbe:
+    enabled: true
+    initialDelaySeconds: 20
+    periodSeconds: 10
+    timeoutSeconds: 5
+    failureThreshold: 6
+    successThreshold: 1
+  ## @param exporter.startupProbe.enabled Enable startupProbe on Harbor Exporter containers
+  ## @param exporter.startupProbe.initialDelaySeconds Initial delay seconds for startupProbe
+  ## @param exporter.startupProbe.periodSeconds Period seconds for startupProbe
+  ## @param exporter.startupProbe.timeoutSeconds Timeout seconds for startupProbe
+  ## @param exporter.startupProbe.failureThreshold Failure threshold for startupProbe
+  ## @param exporter.startupProbe.successThreshold Success threshold for startupProbe
+  ##
+  startupProbe:
+    enabled: false
+    initialDelaySeconds: 5
+    periodSeconds: 10
+    timeoutSeconds: 1
+    failureThreshold: 15
+    successThreshold: 1
+  ## @param exporter.customLivenessProbe Custom livenessProbe that overrides the default one
+  ##
+  customLivenessProbe: {}
+  ## @param exporter.customReadinessProbe Custom readinessProbe that overrides the default one
+  ##
+  customReadinessProbe: {}
+  ## @param exporter.customStartupProbe Custom startupProbe that overrides the default one
+  ##
+  customStartupProbe: {}
+  ## Harbor Exporter containers' resource requests and limits
+  ## ref: https://kubernetes.io/docs/user-guide/compute-resources/
+  ## We usually recommend not to specify default resources and to leave this as a conscious
+  ## choice for the user. This also increases chances charts run on environments with little
+  ## resources, such as Minikube. If you do want to specify resources, uncomment the following
+  ## lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  ## @param exporter.resources.limits The resources limits for the container
+  ## @param exporter.resources.requests The requested resources for the container
+  ##
+  resources:
+    ## Example:
+    ## limits:
+    ##    cpu: 500m
+    ##    memory: 1Gi
+    limits: {}
+    ## Examples:
+    ## requests:
+    ##    cpu: 250m
+    ##    memory: 256Mi
+    requests: {}
```

To be aligned with the other components:
```suggestion
  ## Harbor Exporter resource requests and limits
  ## ref: https://kubernetes.io/docs/user-guide/compute-resources/
  ## @param exporter.resources.limits The resources limits for the Harbor Exporter containers
  ## @param exporter.resources.requests The requested resources for the Harbor Exporter containers
  ##
  resources:
    limits: {}
    requests: {}
```